spec/actions/determine_next_version_using_labels_action_spec.rb

```diff
@@ -0,0 +1,26 @@
+describe Fastlane::Actions::DetermineNextVersionUsingLabelsAction do
+  describe '#run' do
+    let(:mock_repo_name) { 'mock-repo-name' }
+    let(:mock_github_token) { 'mock-github-token' }
+    let(:new_version) { '1.13.0' }
+
+    it 'calls all the appropriate methods with appropriate parameters' do
+      expect(Fastlane::Helper::RevenuecatInternalHelper).to receive(:determine_next_version_using_labels)
+        .with(mock_repo_name, mock_github_token, 3)
+        .and_return(new_version)
+        .once
+
+      Fastlane::Actions::DetermineNextVersionUsingLabelsAction.run(
```

Comment:
You should assert that the value returned by this is `new_version`. You can store this in a variable and do `expect(calculated_version).to eq(new_version)`